aiogram/fsm/middleware.py

```diff
@@ -91,7 +91,7 @@ def get_context(
         self,
         bot: Bot,
         chat_id: int,
-        user_id: int,
+        user_id: Optional[int] = None,
```

Comment:
In case when you use chat_id as user_id in the strategy you don't need to change type-hint here.